template/rating/rating.html

```diff
@@ -1,4 +1,4 @@
 <span ng-mouseleave="reset()" ng-keydown="onKeydown($event)" tabindex="0" role="slider" aria-valuemin="0" aria-valuemax="{{range.length}}" aria-valuenow="{{value}}" aria-valuetext="{{title}}">
     <span ng-repeat-start="r in range track by $index" class="sr-only">({{ $index < value ? '*' : ' ' }})</span>
-    <i ng-repeat-end ng-mouseenter="enter($index + 1)" ng-click="rate($index + 1)" class="glyphicon" ng-class="$index < value && (r.stateOn || 'glyphicon-star') || (r.stateOff || 'glyphicon-star-empty')" ng-attr-title="{{r.title}}"></i>
+    <i ng-repeat-end ng-mouseenter="enter($index + 1)" ng-click="rate($index + 1)" ng-class="(r.iconBaseClass && r.iconBaseClass + ' ' || '') + ($index < value && (r.stateOn || 'glyphicon-star') || (r.stateOff || 'glyphicon-star-empty'))" ng-attr-title="{{r.title}}"></i>
```

Comment:
I'm sorry I don't quite understand the question. I moved the icon base class from class to ng-class so that it would be interpreted correctly. I had to manage the case of undefined icon base class.